### DISCUSSION TITLE
不能做自己喜歡的事怎麼辦？

### DISCUSSION THREAD

**Discussion by PaulSu0905** [1101-GS4719-C/Python-Community-GS4719-C#1287] [Q&A]
我是一個很愛運動的人，尤其是排球還有籃球，現在甚至是系排的隊長
但是因為氣胸復發的關係需要開刀，至少會有一個月的時間不能運動，想到就覺得痛苦，看到其他人能夠一起去打球就很羨慕
想請問大家如果遇到這種事會如何調適心情？
還有可以提供我其他休閒活動讓我能有其他事情做

**Comment by poweichen00** [comment]
我覺得可以培養不一樣的興趣讓這次的低潮度過，相信你可以趕快回到場上的！

**Reply by chia-kai-chang** [reply]
@PaulSu0905 拍拍，我們都支持你～

**Comment by tower20021202** [comment]
我之前手斷掉開刀也有三個月完全不能運動連吃飯都只有一隻手真的很痛苦，但我那段間發現了拼拼圖的樂趣，你可以試試看！

**Comment by Chienyuuu6** [comment]
心有戚戚戚焉

我高中的時候本來以為自己可以在球場上大殺四方的時候，突然膝蓋韌帶就斷掉了，那個時期真的超級黑暗而且很懷疑自己⋯

我已經ㄉ個性偏閒不下來，我能理解被剝奪的難受，但其實隨著長大經歷過很多事件後，開始明白「休息」、「停下來」的重要性。

或許這一次的休息能夠讓你更瞭解自己，你還是可以很喜歡這件事情只是用不一樣的形式擁抱他，這樣的心境思維除了面對現在這件事，其實也能隱約套用在各個層面：課業、人際、未來之類ㄉ

等你回歸自己再辦一個超屌小資盃！！
早日康復，好好休息！

**Reply by chia-kai-chang** [reply]
其實我看到中央的同學都好優秀，我都覺得我輸了～唯一能做的就是「我要放下，不要執著～」

我也常常忙到深夜不休息，其實我們真的應該要如同學所說的，學習「休息」，「停下來」的重要性。

謝謝同學讓我今天體悟了人生的道理

**Comment by Jack-1D** [comment]
ㄨㄚˊ
我前段時間才攔網採到對面的腳翻船，韌帶撕裂。那段時間光是在場下看都覺得好痛苦，好想再跳起來。但後來想想，現在的我所能做到的應該是幫忙訓練球隊練習，而不是每天懊悔當初不應該跳那次攔網的。

**Comment by llh26** [comment]
我覺得你或許可以把這次的療傷時間當成一次休息，去找看看自己有什麼事情之前想做，但可能礙於要練球或做其他的事情還沒探索，這段時間是個好機會！又或許你可以趁著不能打球的時候，偷偷觀察隊員在哪些方面有不足，可以趁這個時候好好觀察，並進一步去做改善～讓團隊可以一起進步，祝Paul Su早日康復！

**Comment by CPW0919** [comment]
高中有一次因為腳傷而錯失上場比賽的機會，當下覺得很難過，後來看到朋友們的好表現後就釋懷了，若新早康復，下次再去跑山!!!

**Comment by harrycoconut** [comment]
兄弟 你可以找我玩😌我就住你隔壁。雖然現在沒法激烈打球，但我相信過一兩個禮拜後，你會找到更棒而且同樣有興趣的事情來做，例如twitch直播或變成真人快打王者，又或是直接飛去曼谷嗨一波，you will find the way😎

**Comment by HsuSungTing** [comment]
我可以理解沒辦法做自己喜歡的事情的難過
我最近發現一個習慣，就是每完成一件小事情(例如洗好碗，或是做完一份作業)就到Daylio這個App記錄下來
我發現每天把那些最然不是特別困難，但完成了之後會對生活有幫助的事情寫下來後
心理會比較可以看報自己目前還能做的事情，會舒服不少
祝你早日康復!!!

**Comment by CCello** [comment]
我認為這是一個發現自己喜歡的新事物的好機會。你可以嘗試學習一些新技能或嗜好，例如烹飪、音樂、攝影、閱讀等，找到自己感興趣的領域，並且深入探索，此外，與朋友和家人交流不論是聊天、晚餐、觀看電影等都是可以舒緩壓力和改善心情的好方法。

**Comment by Chiuhsianglin** [comment]
我前天看羽球亞錦賽麟洋四強的時候，王麒麟在第二局的時候可能大腿拉傷了導致直接結束比賽，當下我的想法是好可惜，差一點就有機會進冠亞了，但是後來我想說健康才是最重要的，之後還有很多機會，好好休息！

**Comment by YoYooHuang** [comment]
換個角度想，有時候休息一段時間反而能讓你更加進步。可以透過這個機會，思考自己的動作哪裡有缺失，也可以看看別人打球，看自己的動作和其他更厲害的人有什麼差別，也許這會是一個轉機。
